File: sk000801/입문/대문자와 소문자.java
class Solution {
    public String solution(String my_string) {
        String s = "";
        for(int i = 0; i < my_string.length(); i++) {
            char c = my_string.charAt(i);
            if(Character.isUpperCase(c)) s += Character.toString(Character.toLowerCase(c));
            else s += Character.toString(Character.toUpperCase(c));
        }
        return s;
    }
}

File: kyw0716/level0/대문자와 소문자.js
function solution(my_string) {
  return my_string
    .split("")
    .map((v) => {
      if (v.match(/[a-z]/) !== null) return v.toUpperCase();
      if (v.match(/[A-Z]/) !== null) return v.toLowerCase();
    })
    .join("");
}

// https://school.programmers.co.kr/learn/courses/30/lessons/120893
